boards/arm/nucleo_g071rb/nucleo_g071rb.dts

```diff
@@ -22,7 +23,7 @@
 	leds {
 		compatible = "gpio-leds";
 		green_led_1: led_4 {
-			gpios = <&gpioa 5 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpioa 5 GPIO_ACTIVE_LOW>;
```

Comment:
According to [STM32 Nucleo-64 boards with STM32G07xRB MCUs](https://www.st.com/content/ccc/resource/technical/document/user_manual/group1/5e/a5/b2/30/af/d4/45/d0/DM00452640/files/DM00452640.pdf/jcr:content/translations/en.DM00452640.pdf)
```
LD4 USER: this green LED is a user LED connected to Arduino signal D13 corresponding to STM32 I/O PA5. To light the LED a high-logic state ‘1’ has to be written in the corresponding GPIO.
```